docker-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+BOOTSTRAP_DIR="${HOME}/bootstrap"
+
+# Make sure we can see rbenv
+eval "$(~/.rbenv/bin/rbenv init - bash)"
+
+# Pull in fonts for theme
+rsync -av $BOOTSTRAP_DIR/fonts/ ./fonts/
+cp ${BOOTSTRAP_DIR}/Gemfile.lock ./
+
+# Execute build
+bundle exec rake "$@"
```

Comment:
`./build-docker.sh` ran up until this `bundle exec` line and then failed with
```
rbenv: version `3.1.2' is not installed (set by /workspace/.ruby-version)
```

Almann pointed out that it was pulling in my `.ruby-version` from my previous non-docker build.  `rm .ruby-version` fixed the issue.